initfiles/bash/etc/systemd/system/CMakeLists.txt

```diff
@@ -29,7 +29,9 @@ set(componentList
     sasha
     thor
     thorslave
-    backupnode)
+    backupnode
+    spark
```

Comment:
why do we have two spark related components listed here, and should they be renamed?